pkg/http/http_test.go

```diff
@@ -59,6 +62,8 @@ func TestNewInstrumentedClient(t *testing.T) {
 }
 
 func TestPathProcessor(t *testing.T) {
```

Comment:
We have some tests with t.Paralel, but there are no benefits at the moment, and only issues. Example https://github.com/kubernetes-sigs/prow/issues/450

Basically we have to execute tests not just on Github Runners, but on kubernetes infra, and resources are limited there.

 Was thinking about mandating it a while ago with linter https://golangci-lint.run/usage/linters/#paralleltest, 
but resource constraints would not let us do it.

Reply:
Ah, good to know. I've removed the new t.Parallel()s